docs/public/installation/zonemaster-backend.md

```diff
@@ -111,9 +111,9 @@ cd `perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")'`
 sudo install -v -m 755 -d /etc/zonemaster
 sudo install -v -m 640 -g zonemaster ./backend_config.ini /etc/zonemaster/
 sudo install -v -m 775 -g zonemaster -d /var/log/zonemaster
-sudo install -v -m 755 ./zm-rpcapi.lsb /etc/init.d/zm-rpcapi
-sudo install -v -m 755 ./zm-testagent.lsb /etc/init.d/zm-testagent
 sudo install -v -m 755 ./tmpfiles.conf /usr/lib/tmpfiles.d/zonemaster.conf
+sudo install -v -m 664 ./zm-rpcapi.service /etc/systemd/system/

```

Comment:
Wouldn’t 644 be a better file mode than 664, in order to mirror the 755 mode formerly used for the init scripts?

Reply:
I just followed the [documentation](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_working-with-systemd-unit-files_configuring-basic-system-settings#proc_creating-custom-unit-files_assembly_working-with-systemd-unit-files). It looked funny to me too but I thought maybe that there's a reason for doing things that way. But it's probably a typo in the documentation. I discussed this with a couple of ops people and they agreed with the typo hypothesis.